tests/features/steps/basic_commands.py

```diff
@@ -32,6 +33,19 @@ def step_see_list_databases(context):
 def step_run_cli(context):
     wrappers.run_cli(context)
 
+@when('we launch dbcli using {arg}')
+def step_run_cli_using_arg(context, arg):
+    prompt_check = False
+    if arg == '--username':
```

Comment:
Looks like you're not doing anything special for `arg != "password"`. Could combine that into a default case.

Reply:
`--password` will prompt for the password which is different behavior than just simply passing in `--username`.

Reply:
I meant that while `--password` and `dsn_password` look like your special cases, the rest of the args are not so special. So you could probably do something like:

```
if arg == "--password":
   # do my special thing
elif arg == "dsn_password":
   # another special thing
else:
   arg = "{}={}".format(arg, context.config[arg.replace('--', '')])
```